packages/account/src/account.ts

```diff
@@ -63,6 +63,8 @@ export type EstimatedTxParams = Pick<
 >;
 const MAX_FUNDING_ATTEMPTS = 2;
 
+const MAX_TRANSACTION_INPUTS = 255;
+
```

Comment:
We cannot use the defined constant here because this value is set on the chain. So the right approach would be to read this value from the chain.

Reply:
https://github.com/FuelLabs/fuels-ts/pull/2822/commits/9535163ccfc0677c019ede3d65536df9c54639a9